app/storyedit/directives/edit.component.ts

```diff
@@ -1,62 +1,59 @@
+import {CATEGORIES, SUBCATEGORIES} from '../models/story.categories';
+import {StoryFieldComponent} from './storyfield.component';
 
 @Component({
-  directives: [NgForm],
+  directives: [StoryFieldComponent],
   selector: 'newstory-edit',
-  styleUrls: ['app/storyedit/storyedit.forms.css'],
+  styleUrls: ['app/storyedit/directives/edit.component.css'],
   template: `
     <form *ngIf="story && story.isLoaded">
 
-      <div [class]="fieldClasses('title')">
-        <h3><label for="title" required>Story Title</label></h3>
-        <p class="hint">Write a short, Tweetable title like a newspaper headline.
-          Make viewers want to click on your piece from the title alone.</p>
-        <textarea id="title" [(ngModel)]="story.title"></textarea>
-        <p class="error">Title {{story.invalid.title}}</p>
-      </div>
-
-      <div [class]="fieldClasses('shortDescription')">
-        <h3><label for="teaser" required>Teaser</label></h3>
-        <p class="hint">A first impression; think of this as the single-item
-          lead of a piece.</p>
-        <textarea id="teaser" [(ngModel)]="story.shortDescription"></textarea>
-        <p class="error">Teaser {{story.invalid.shortDescription}}</p>
-      </div>
+      <story-field [story]="story" textarea="true" name="title" label="Story Title" required>
+        <hint>Write a short, Tweetable title like a newspaper headline. Make viewers
+          want to click on your piece from the title alone.</hint>
+      </story-field>
+
+      <story-field [story]="story" textarea="true" name="shortDescription" label="Teaser" required>
+        <hint>A first impression; think of this as the single-item lead of a piece.</hint>
+      </story-field>
 
       <hr/>
 
-      <div class="field">
-        <h3><label required>Story Versions</label></h3>
+      <story-field label="Story Versions" required>
         <div class="uploads-box">uploads goes heres</div>
-      </div>
+      </story-field>
 
       <hr/>
 
-      <div class="field">
-        <h3><label required>Tag your Story</label></h3>
-        <p class="hint">Adding tags that are relevant to your piece helps people
-          find your work on PRX and can help you be licensed and heard.</p>
-      </div>
+      <story-field [story]="story" label="Tag your Story" invalid="tags" invalidlabel="" required>
+        <hint>Adding tags that are relevant to your piece helps people find your work
+          on PRX and can help you be licensed and heard.</hint>
+        <div class="span-fields">
+          <story-field [story]="story" [select]="GENRES" name="genre" label="Genre"
+            small="true" required></story-field>
+          <story-field [story]="story" [select]="SUBGENRES" name="subGenre" label="SubGenre"
+            small="true" required></story-field>
+        </div>
+        <story-field [story]="story" textinput="true" name="extraTags"
+          label="Extra Tags" small="true"></story-field>
+      </story-field>
```

Comment:
This is just an opinion, but I am unsure how much tagging like this is helpful, since the only thing itunes and others care about are the tags at the podcast/channel level, not the individual story/item level.

The main time these are useful is when a story isn't in a series/podcast, so I am not gonna sweat them too much at the moment.